pkgs/desktops/plasma-5/plasma-nm/default.nix

```diff
@@ -14,6 +14,10 @@ plasmaPackage {
       src = ./0001-mobile-broadband-provider-info-path.patch;
       inherit mobile_broadband_provider_info;
     })
+    (substituteAll {
+      src = ./0002-openvpn-binary-path.patch;
+      inherit openvpn;
+    })
```

Comment:
OK - which one is it going to be?  The only approach that I know will reliably work is patching the source.  If updating the patch becomes irritating over time, maybe we can find a way to do it with sed or something instead of a patch?